### DISCUSSION TITLE
How do I update my Uptime Kuma Dashboard?

### DISCUSSION THREAD

**Discussion by sysmin1** [themepark-dev/theme.park#656] [Q&A] [answered]
This is the first time I'm trying these themes, sorry about the noob question. I was able to copy the CSS code to the Custom CSS field in my status page, and it works. How do I do the same for the main dashboard?

It's a self-hosted instance running in Debian 12 as a LXC (Linux container), [installed through a script from this site](https://tteck.github.io/Proxmox/).

**Comment by GilbN** [comment, marked as answer]
Have a read here: https://docs.theme-park.dev/setup/#how-the-hell-do-i-add-a-theme-to-my-app

**Comment by sysmin1** [comment]
So all methods need an external setup to inject the theme? I do have Nginx Proxy Manager running, but would prefer a native solution. Poking the code didn't work (yet!), and this may be a better question for the Uptime Kuma team.

Anyways, thank you, have a nice day :)

**Reply by GilbN** [reply]
No, some need subfilter, not all.

**Reply by sysmin1** [reply]
I understand. Checking with the Uptime Kuma team [over here](https://github.com/louislam/uptime-kuma/issues/5218), it seems that native theming is partially supported, which is good enough for now. Thanks for your support,
